lib/posix/pthread.c

```diff
@@ -534,6 +606,21 @@ int pthread_attr_setschedpolicy(pthread_attr_t *attr, int policy)
 	return 0;
 }
 
+/**
+ * @brief Set stack size attribute in thread attributes object.
+ *
+ * See IEEE 1003.1
+ */
+int pthread_attr_setstacksize(pthread_attr_t *attr, size_t stacksize)
+{
+	if ((attr == NULL) || (attr->initialized == 0U)) {
+		return EINVAL;
+	}
```

Comment:
```suggestion
	if ((attr == NULL) || (attr->initialized == 0U) || (stacksize < ARCH_MIN_STACK_SIZE)) {
		return EINVAL;
	}
```
I think this should make sure the stacksize is at least ARCH_MIN_STACK_SIZE aswell

Reply:
I think that change is already in

Reply:
Sorry - my mistake. I'll make that change - thanks!